### ISSUE TITLE
Clarification on prelim instructions

### ISSUE THREAD

**Issue by robfairh** [arfc/arfc.github.io#270]
## I'm submitting a ...
  - [ ] bug report
  - [x] feature request
  - [ ] question

## Expected Behavior
The [prelim instructions](http://arfc.github.io/manual/guides/writing/prelim) should mention that there is an NPRE committee requirement.

## Actual Behavior
The website says that the NPRE rules are written on the grad school handbook.
This is not completely accurate as NPRE has rules that build on top of the [grad school handbook](https://grad.illinois.edu/sites/grad.illinois.edu/files/pdfs/handbook.pdf) rules (see Dissertation Committee in the handbook).

## How can this issue be closed?
Adding the NPRE requirement.
This is actually the requirement that matters, so referring the students to the handbook is a bit of a waste of time (IMO).
Long story short, the NPRE requirement is to have:
* 3 people (faculty/research scientist) from NPRE
* 1 person from UIUC outside NPRE

There can be more people than this (which may include people outside UIUC), but this the minimum requirement.
